editions/tw5.com/tiddlers/mechanisms/InfoMechanism.tid

```diff
@@ -10,6 +10,10 @@ type: text/vnd.tiddlywiki
 
 System tiddlers in the namespace `$:/info/` are used to expose information about the system (including the current browser) so that WikiText applications can adapt themselves to available features.
 
+! Important
+
+The values below are "snapshots". The values are initialized at the wiki startup.
```

Comment:
Not entirely. I think that the only other values that can change without the wiki being reloaded are the screensize.

Reply:
hmm. where is my reply. click here: https://github.com/Jermolene/TiddlyWiki5/pull/4374#discussion_r351220202